### ISSUE TITLE
Plex MPV Shim

### ISSUE THREAD

**Issue by matthew-piziak** [NixOS/nixpkgs#66808]
**Project description**
This project aims to allow casting of content from a Plex server to MPV, with minimal dependencies to prevent the project from becoming unmaintained.

Plex Media Player is slated to become unmaintained, so a good replacement is in order.

**Metadata**

* homepage URL: https://github.com/iwalton3/plex-mpv-shim
* source URL: https://github.com/iwalton3/plex-mpv-shim
* license: none visible
* platforms: Unix

**Comment by matthew-piziak** [comment]
I attempted the following derivation:
```
{ config, pkgs, ... }:
let
  plex-mpv-rev = "6ecc912a4f41586719c3430870db4004022a30a7";
  plex-mpv-shim = pkgs.python3Packages.buildPythonApplication rec {
    pname = "plex-mpv-shim";
    version = "0.1-${plex-mpv-rev}";
    name = "${pname}-${version}";
    src = pkgs.fetchFromGitHub {
      owner = "iwalton3";
      repo = "plex-mpv-shim";
      rev = plex-mpv-rev;
      sha256 = "1kh4p91sf7yxdrkrwfhd2wj5xzkssixxpb3y89c9ybf6wdr00lpz";
    };
    propagatedBuildInputs = with pkgs.python3Packages; [ mpv requests ];
  };
in {
  environment.systemPackages = [ plex-mpv-shim ];
}
```

Unfortunately it fails with the following error: 
```
Processing ./Plex_MPV_Shim-0.2-py3-none-any.whl
Requirement already satisfied: requests in /nix/store/qssdbbm50hxjsd59ivbx5ml0iahb49wd-python3.7-requests-2.21.0/lib/python3.7/site-packages (from Plex-MPV-Shim==0.2) (2.21.0)
Collecting python-mpv (from Plex-MPV-Shim==0.2)
  Could not find a version that satisfies the requirement python-mpv (from Plex-MPV-Shim==0.2) (from versions: )
No matching distribution found for python-mpv (from Plex-MPV-Shim==0.2)
builder for '/nix/store/6mf4skgqq8zvgdl2k4affssj5sx6xigc-plex-mpv-shim-0.1-6ecc912a4f41586719c3430870db4004022a30a7.drv' failed with exit code 1
```

Assistance requested.

**Comment by matthew-piziak** [comment]
This requires `mpv 0.2` but `nixpkg` only has `0.1`.

**Comment by averelld** [comment]
This is closed by #77884
